config/routes.rb

```diff
@@ -24,6 +24,7 @@
     root to: 'legal_aid_applications#index'
     resources :legal_aid_applications, only: [:index] do
       delete :destroy_all, on: :collection
+      delete :destroy
```

Comment:
instead of that line, would adding `destroy` to `only: [:index, :destroy]` (line 25) work ?
Then you would get the id of the application via `params[:id]` instead `params[:legal_aid_application_id]`

I think that would be the proper 'railsy' way to do

Reply:
Yes - that's a much better way of doing it!